disk-buffering/build.gradle.kts

```diff
@@ -20,7 +20,7 @@ dependencies {
   api("io.opentelemetry:opentelemetry-sdk")
   compileOnly("com.google.auto.value:auto-value-annotations")
   annotationProcessor("com.google.auto.value:auto-value")
-  signature("com.toasttab.android:gummy-bears-api-24:0.5.1@signature")
+  signature("com.toasttab.android:gummy-bears-api-24:0.6.1@signature")
```

Comment:
@LikeTheSalad @breedx-splk should we drop this to `gummy-bears-api-21` based on discussion in https://github.com/open-telemetry/opentelemetry-java/issues/5936?

Reply:
Yeah, it makes sense, I think we should use 21 here as well. I'll create a PR for it.